netbout-web/src/main/java/com/netbout/email/EmAlias.java

```diff
@@ -145,20 +140,31 @@ public void email(final String email, final String link)
                     )
                     .with(
                         new EnHTML(
-                            Joiner.on('\n').join(
+                            StringUtils.join(
```

Comment:
@essobedo I uset StringUtils because I did not neet the ``on(...)`` method anymore, and it seems more readable than ``Joiner``.

I will do the rest of the changes you mentioned later today or tomorrow morning